.github/workflows/.deploy.yml

```diff
@@ -0,0 +1,154 @@
+        description: GitHub/OpenShift environment; usually PR number, test or prod
+        default: 'dev'
+        required: false
+        type: string
+      tag:
+        description: Container tag; usually PR number
+        default: ${{ github.event.number }}
+        required: false
+        type: string
+      target:
+        description: Deployment target; usually PR number, test or prod
+        default: ${{ github.event.number }}
+        required: false
+        type: string
+
+permissions: {}
+
+jobs:
+  init:
+    name: Deploy (init)
+    environment: ${{ inputs.environment }}
+    runs-on: ubuntu-24.04
+    outputs:
+      url: ${{ steps.calculate.outputs.url }}
+    steps:
+      # steps.calculate.outputs.url => needs.init.outputs.url
+      - name: Calculate the deployment number
+        id: calculate
+        run: |
+          echo "url=${{ github.event.repository.name }}-$((${{ github.event.number }} % 50))-frontend.apps.silver.devops.gov.bc.ca" >> $GITHUB_OUTPUT
+          echo "url=${{ github.event.repository.name }}-$((${{ github.event.number }} % 50))-frontend.apps.silver.devops.gov.bc.ca"
+
+      - name: Deployment Init
+        uses: bcgov/action-deployer-openshift@v4.0.0
+        with:
+          file: common/openshift.init.yml
+          oc_namespace: ${{ secrets.OC_NAMESPACE }}
+          oc_server: ${{ secrets.oc_server }}
+          oc_token: ${{ secrets.oc_token }}
+          overwrite: true
+          parameters:
+            -p ZONE=${{ inputs.target }}
+            -p ORACLEDB_USER=${{ secrets.ORACLEDB_USERNAME }}
+            -p ORACLEDB_PASSWORD=${{ secrets.ORACLEDB_PASSWORD }}
+            -p ORACLEDB_USER_W=THE
+            -p ORACLEDB_PASSWORD_W=${{ secrets.ORACLEDB_PASSWORD_W }}_${{ github.event.number }}
+            -p ORACLEDB_DATABASE=${{ secrets.ORACLEDB_DATABASE }}
+            -p ORACLEDB_HOST="nr-forest-client-tools-legacydb.d2723f-tools.svc.cluster.local"
+            -p ORACLEDB_SERVICENAME=PR_${{ github.event.number }}
+            -p ORACLEDB_SECRET=${{ secrets.ORACLEDB_SECRET }}
+            -p BCREGISTRY_KEY=${{ secrets.BCREGISTRY_KEY }}
+            -p BCREGISTRY_ACCOUNT=${{ secrets.BCREGISTRY_ACCOUNT }}
+            -p CHES_CLIENT_ID=${{ secrets.CHES_CLIENT_ID }}
+            -p CHES_CLIENT_SECRET=${{ secrets.CHES_CLIENT_SECRET }}
+            -p ADDRESS_COMPLETE_KEY=${{ secrets.ADDRESS_COMPLETE_KEY }}
+            -p COGNITO_ENVIRONMENT=DEV
+            -p CHES_MAIL_COPY=${{ secrets.CHES_MAIL_COPY }}
+            -p VITE_NODE_ENV=openshift-dev
+            -p URL_FRONTEND=${{ steps.calculate.outputs.url }}
+            -p GREEN_DOMAIN=${{ secrets.GREEN_DOMAIN }}
+            -p COGNITO_CLIENT_ID=${{ secrets.COGNITO_CLIENT_ID }}
+            -p COGNITO_DOMAIN=${{ secrets.COGNITO_DOMAIN }}
+            -p COGNITO_REGION=${{ secrets.COGNITO_REGION }}
+            -p COGNITO_USER_POOL=${{ secrets.COGNITO_USER_POOL }}
+            -p LANDING_URL=${{ steps.calculate.outputs.url }}
```

Comment:
This is only valid for the PR environment, but test and prod require the vanity URL

Reply:
Addressed.  This PR will only go to dev PR enironments for now.  Please expect follow up soon.